src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -33,12 +34,18 @@ internal TagHelperOutput(string tagName)
         /// <param name="tagName">The HTML element's tag name.</param>
         /// <param name="attributes">The HTML attributes.</param>
         public TagHelperOutput(string tagName, [NotNull] IDictionary<string, string> attributes)
+            : this(tagName, attributes, HtmlEncoder.Default)
```

Comment:
I'd rather make the API simple (I like it as is), only time users will care about providing an HTMLEncoder is if they've already registered one themselves therefore it makes sense for them to pull in their own and provide it.